modules/sdk-coin-world/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,79 @@
+import { getBuilder } from '../getBuilder';
+import should from 'should';
+import { TransactionType } from '@bitgo/sdk-core';
+import { decodeTransferData } from '@bitgo/abstract-eth';
+import { TransactionBuilder } from '../../../src';
+import * as testData from '../resources';
+
+describe('World Transfer Builder', () => {
+  describe('Build from TxHex', function () {
+    it('Should successfully build from txHex', async function () {
+      const txBuilder = getBuilder('tworld');
+      const txHex = testData.TXN_Hash;
+      txBuilder.from(txHex);
+      const parsedTx = await txBuilder.build();
+
+      should.exist(parsedTx.toJson());
+    });
+  });
+});
+
+describe('World transaction builder send', () => {
+  describe('should sign and build', () => {
+    let txBuilder;
+    let key;
+    let contractAddress;
+
+    beforeEach(() => {
+      contractAddress = testData.CONTRACT_ADDR;
+      txBuilder = getBuilder('tworld') as TransactionBuilder;
+      key = testData.PRIVATE_KEY_1;
+      txBuilder.fee({
+        fee: '1000000000',
+        gasLimit: '12100000',
+      });
+      txBuilder.counter(2);
+      txBuilder.type(TransactionType.Send);
+      txBuilder.contract(contractAddress);
+    });
+
+    it('a send funds transaction', async () => {
+      const recipient = testData.RECIPIENT;
+      const amount = '1000000000';
+      const expireTime = 1590066728;
+      const sequenceId = 5;
+      txBuilder
+        .transfer()
+        .amount(amount)
+        .to(recipient)
+        .expirationTime(expireTime)
+        .contractSequenceId(sequenceId)
```

Comment:
This is not multisig wallet, why are we needing expirationTime, contractSequenceId, contract